TLM/TLM/U/Panel/BaseUWindowPanel.cs

```diff
@@ -45,13 +45,6 @@ public abstract class BaseUWindowPanel
         /// <summary>Called by UResizer for every control after it is to be 'resized'.</summary>
         public virtual void OnAfterResizerUpdate() { }
 
-        /// <summary>Called by UnityEngine when component gets destroyed</summary>
-        public override void OnDestroy() {
-            uiScaleUnbsubscriber_.Dispose();
-            uiTransparencyUnbsubscriber_.Dispose();
-            base.OnDestroy();
-        }
-
         /// <summary>Invoke rescaling handler, because possibly it has the new size now.</summary>
```

Comment:
Why removed?

Reply:
Asssumed garbage collector will eventually pick these up. Or do we have to do explicit destructions now?

Reply:
Reverted